docs/Contributors_Guide/github_workflow.rst

```diff
@@ -305,9 +305,9 @@ Open a pull request using a browser
       reviewer, internal contributors submitting pull requests should select
       the appropriate reviewer(s) and let the reviewer know that the pull
       request has been assigned to them. If external contributors are unsure
-      who to assign as a reviewer, they should send an email to
-      met_help@ucar.edu asking the met-help point person to help with the
-      assignment of a reviewer.
+      who to assign as a reviewer, create an issue in the 
```

Comment:
```suggestion
      who to assign as a reviewer, create a post in the 
```
I suggest this is changed to be consistent with the wording in the other mentions of the discussions forum.

Reply:
Great catch!  I'm not sure why I wrote issue... It definitely should be post.  I'll update that now.